lib/string/sprintf.c

```diff
@@ -17,3 +17,8 @@
 extern inline int xasprintf(char **restrict s, const char *restrict fmt, ...);
 extern inline int xvasprintf(char **restrict s, const char *restrict fmt,
     va_list ap);
+
+extern inline int snprintf_(char *restrict s, int size,
+    const char *restrict fmt, ...);
+extern inline int vsnprintf_(char *restrict s, int size,
+    const char *restrict fmt, va_list ap);
```

Comment:
The function `vsnprintf` takes `size_t` for `size` and `vsnprintf_` should do the same.